tests/toDo.test.js

```diff
@@ -0,0 +1,72 @@
+import ToDo from "../src/models/toDo";
+import ToDoList from "../src/models/toDoList";
+import { describe, it, expect } from "vitest";
+
+describe("ToDo", () => {
+  it("should create a new ToDo", () => {
+    const form = document.createElement("form");
+    const input = document.createElement("input");
+    input.name = "description";
+    input.value = "Buy milk";
+    form.appendChild(input);
+
+    const toDo = new ToDo(form);
+
+    expect(toDo.id).toBeGreaterThan(0);
+    expect(toDo.description).toBe("Buy milk");
+    expect(toDo.done).toBe(false);
+    expect(toDo.createdAt).toBeInstanceOf(Date);
+  });
+
+  it("should update a ToDo", () => {
+    const form = document.createElement("form");
+    const input = document.createElement("input");
+    input.name = "description";
+    input.value = "Buy milk";
+    form.appendChild(input);
+
+    const toDo = new ToDo(form);
+
+    const newForm = document.createElement("form");
+    const newInput = document.createElement("input");
+    newInput.name = "description";
+    newInput.value = "Buy eggs";
+    newForm.appendChild(newInput);
+
+    toDo.update(newForm);
+
+    expect(toDo.description).toBe("Buy eggs");
+  });
+
+  it("should create an element", () => {
+    const form = document.createElement("form");
+    const input = document.createElement("input");
+    input.name = "description";
+    input.value = "Buy milk";
+    form.appendChild(input);
+
+    const toDo = new ToDo(form);
```

Comment:
```suggestion
    const form = document.createElement("form");
    const input = document.createElement("input");
    input.name = "description";
    input.value = "Buy milk";
    form.appendChild(input);

    const toDo = new ToDo(form);
```

@sal-wtag vai, we can deal with this like shown in previous suggestion